.travis.yml

```diff
@@ -8,51 +8,104 @@ addons:
       - mysql-client-core-5.6
       - mysql-client-5.6
       - postfix
-  firefox: latest
+  chrome: stable
   hosts:
     - magento2.travis
 services:
   - rabbitmq
   - elasticsearch
+  - mysql
 language: php
 php:
   - '7.1'
   - '7.2'
   - '7.3'
 env:
   global:
-    - COMPOSER_BIN_DIR=~/bin
+    # we should revisit the nodejs version. Long Term Stable (LTS) is version 12 now.
     - NODE_JS_VERSION=8
```

Comment:
@sivaschenko what do you think of bumping the nodejs version to 12? What do we use node for, anyways?

Reply:
I think we used it for eslint tests, as we are running them on Jenkins now, I think we can remove node dependency